File: src/golang/fizzbuzz.go
package main

import (
	"fmt"
)

func fizzbuzz() {
	for num := 1; num <= 20; num++ {
		var fizz = num%3 == 0
		var buzz = num%5 == 0
		if fizz && buzz {
			fmt.Println("fizzbuzz")
		} else if fizz {
			fmt.Println("fizz")
		} else if buzz {
			fmt.Println("buzz")
		} else {
			fmt.Println(num)
		}
	}
}

func main() {
	fizzbuzz()
}

File: src/python/fizzbuzz.py
def fizzbuzz():
    for num in range(1, 21):
        fizz = num % 3 == 0
        buzz = num % 5 == 0
        if fizz and buzz:
            print("fizzbuzz")
        elif fizz:
            print("fizz")
        elif buzz:
            print("buzz")
        else:
            print(num)


if __name__ == '__main__':
    fizzbuzz()
